api_app/connectors_manager/helpers.py

```diff
@@ -2,15 +2,45 @@
 # See the file 'LICENSE' for copying permission.
 
 from django.conf import settings
+from django.core.cache import cache
 
 import os
 import json
+import logging
 
+from api_app.connectors_manager.serializers import ConnectorConfigSerializer
 
-def get_connector_config():
+
+logger = logging.getLogger(__name__)
+
+
+def get_verified_connector_config():
+    # get cached json else cache and return
+    config = cache.get("verified_connectors_config", None)
+    if config is not None:
+        return config
+    else:
+        success, config = verify_connector_config()
+        if success:
+            cache.set("verified_connectors_config", config)
+        return config
+
+
+def verify_connector_config():
```

Comment:
I'd prefer this method to be under the `ConnectorConfigSerializer` as a `@classmethod` with name `read_and_verify_config`. Then we can do something like: `ConnectorConfigSerializer.read_and_verify_config()`; 

I mean, it's just better to keep related code together in one class.